app/views/actors/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<p id="notice"><%= notice %></p>
+
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th></th>
+        <th colspan="2"></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @actors.each do |actor| %>
+          <tr>
+            <td>
+              <% if actor.image.attached? %>
+                <%= image_tag actor.image %>
+              <% else %>
+                <%= image_tag("default_movie_image.png") %>
+              <% end %>
+            </td>
+            <td><%= link_to actor.name, actor %></td>
+            <td><%= link_to 'Edit', edit_actor_path(actor), class: "btn btn-primary" %></td>
+            <td><%= link_to 'Destroy', actor, class: "btn btn-primary", method: :delete, data: { confirm: 'Are you sure?' } %></td>
+          </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <br>
+
+  <% if current_user_admin? %>
```

Comment:
Typo here...also this means that you didn't cover this edge case inisde the tests